AmplifyPlugins/API/AWSAPICategoryPluginFunctionalTests/GraphQLSyncBased/GraphQLSyncBasedTests.swift

```diff
@@ -48,12 +48,17 @@ class GraphQLSyncBasedTests: XCTestCase {
         let uuid = UUID().uuidString
         let testMethodName = String("\(#function)".dropLast(2))
         let title = testMethodName + "Title"
-        guard let post = createPost(id: uuid, title: title) else {
+        guard let createMutation = createPost(id: uuid, title: title) else {
```

Comment:
Is `createPost` creating a Post model or a CreatePost mutation? Please align the names appropriately

Reply:
updating to `createdPost`